src/matchmaking/match/match.hpp

```diff
@@ -17,10 +17,16 @@ class DrawTracker {
         draw_score   = tournament_config.draw.score;
     }
 
-    void update(const int score, const int move_count, engine::ScoreType score_type) noexcept {
+    void update(const int score, const int move_count, engine::ScoreType score_type, chess::Color color,
+               std::string Fen) noexcept {
```

Comment:
why get the fen only to split the fen and parse it? call the according functions on the board object to get the hmvc

Reply:
> why get the fen only to split the fen and parse it? call the according functions on the board object to get the hmvc

i see, whats the function to get the hmvc from the board object?

Reply:
well im using my chess lib which got it's docs here https://disservin.github.io/chess-library/pages/usage.html